nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -755,18 +735,17 @@ in
           '';
       };
 
-    virtualisation.efiVars =
```

Comment:
Please add a `mkRemovedOptionModule` for this option.
Alternatively, if this option could be important to users, you could add options that replicate the defaulting logic in the `make-disk-image.nix` parameters and use `mkRenamedOptionModule`.